html-tables/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> HTML TABLES </title>
+</head>
+
+<body>
+
+  <h1> Student Grade Table </h1>
+
+  <table>
+
```

Comment:
Almost there! The only thing to add is the `thead` and `tbody` elements for the headings and data

Reply:
done!